enclave_manager/avalon_enclave_manager/wpe/wpe_requester.py

```diff
@@ -286,7 +287,12 @@ def _post_and_get_result(self, json_rpc_request):
         """
         json_request_str = json.dumps(json_rpc_request)
         logger.info("Request to KME listener %s", json_request_str)
-        response = self._uri_client._postmsg(json_request_str)
+        try:
+            response = self._uri_client._postmsg(json_request_str,
```

Comment:
Add a comment why WPE should retry. Something like "KME listener may not have started and not ready to accept request from WPE. Retry sending the request "